src/components/Checkbox/Checkbox.tsx

```diff
@@ -17,6 +18,8 @@ export interface CheckboxProps extends TPAComponentProps {
   indeterminate?: boolean;
   error?: boolean;
   name?: string;
+  box?: boolean;
```

Comment:
In order to align with other components (radioButton for one), instead of having a `box` prop, please use a `theme` prop.
You can consult your fellow teammate @mariakwix on the prop she's adding for the RadioButton component